resolwe/flow/management/commands/register.py

```diff
@@ -319,3 +319,5 @@ def handle(self, *args, **options):
                 self.find_schemas(desc_path, filters=schemas, schema_type='descriptor', verbosity=verbosity))
 
         self.register_descriptors(descriptor_schemas, user_admin, force, verbosity=verbosity)
+
+        manager.get_executor().post_register_hook()
```

Comment:
Output some text to note what is happening (e.g., "Running executor post-registration hook...").

Reply:
Done! :)